.github/workflows/_docker-publish.yml

```diff
@@ -7,6 +7,8 @@ on:
       - "Cargo.toml"
       - ".github/workflows/rust.yml"
       - ".github/workflows/docker-publish.yml"
+    types: [review_requested, ready_for_review]
```

Comment:
Should we make it only ready for review, so that a draft where some are requested to look doesn't get run before marking ready?

Reply:
Yeah that's a good point. Updating.

Reply:
seems like it's not how we expected, prob should revert